indico/htdocs/js/indico/Core/Dialogs/Users.js

```diff
@@ -1615,12 +1637,13 @@ type("UserListField", ["IWidget"], {
      */
     function(userDivStyle, userListStyle,
              initialUsers, includeFavourites, suggestedUsers,
-             allowSearch, enableGroups, conferenceId, privileges,
+             allowSearch, enableGroups, enableIpNetworks, conferenceId, privileges,
```

Comment:
Yup I don't like it either. However I wanted to group more or less related options together